app/uk/gov/hmrc/gform/formtemplate/FormComponentMakerService.scala

```diff
@@ -63,11 +64,14 @@ object FormComponentMakerService {
     maybeValueExpr: Option[ValueExpr],
     displayWidth: Option[String],
     multiLine: Option[String],
+    rows: Option[Int],
```

Comment:
This may be already an `Int`. 

It will avoid repetition of `rows.getOrElse(TextAreaRows.default)` down the line.

Reply:
Perfect. Done.